Tests/TuistDependenciesTests/SwiftPackageManager/Utils/PackageInfoMapperTests.swift

```diff
@@ -1598,6 +1639,7 @@ extension PackageInfo.Target {
         name: String,
         type: PackageInfo.Target.TargetType = .regular,
         path: String? = nil,
+        url: String? = nil,
```

Comment:
what about getting an `URL` and putting it in path or url, depending on the type of url?
Otherwise you might define one with both `path` and `url` which is not a valid scenario

Reply:
can't do because init has also both properties, and it's only in tests. But agree, feel free to change.